src/sql/query/extract_employee_data_from_raw_data.sql

```diff
@@ -0,0 +1,26 @@
+SELECT
+employee_id AS client_employee_id,
+INITCAP(first_name) AS first_name,
+INITCAP(last_name) AS last_name,
+d.client_department_id AS department_id,
+(CASE WHEN manager_employee_id = '-' 
+ THEN NULL
+ ELSE manager_employee_id END) AS manager_employee_id,
+CAST(salary AS FLOAT) AS salary,
+CAST(hire_date as DATE),
+CAST(
+	CASE WHEN terminated_date = '01-01-1700' 
+	THEN NULL
+	ELSE terminated_date END AS DATE
+),
```

Comment:
```suggestion
  (CASE
    WHEN terminated_date = '01-01-1700' 
      THEN NULL
     ELSE CAST(terminated_date AS DATE) 
   END) AS terminated_date,
```

Reply:
Why are doing this tranformation?

Reply:
Didi yo chahi because hamro file ma terminated_date ko values are in this format '01-01-1700' for the employees who haven't been terminated, so we put it null, ani bhayeko haru ko chahi type cast to date.